features/FxA-53-send-sms/README.md

```diff
@@ -0,0 +1,444 @@
+  </tr>
+  <tr>
+    <td>flow.sms.link.why</td>
+    <td># of people that click on “why is this required”</td>
+  </tr>
+  <tr>
+    <td>flow.sms.link.maybe_later</td>
+    <td># of people that click “maybe later”</td>
+  </tr>
+</table>
+
+
+### Formula
+
+<table>
+  <tr>
+    <td>SMS form submit rate</td>
+    <td>flow.sms.submit / flow.sms.view</td>
+  </tr>
+  <tr>
+    <td>SMS form submit success rate</td>
+    <td>sms.1.sent / flow.sms.submit </td>
+  </tr>
+  <tr>
+    <td>% of people that sign in that have gone through this flow</td>
+    <td></td>
+  </tr>
+</table>
+
+
+* * *
+
+
+## **Detailed design**
+
+### Audience
+
+This test will target:
+
+* USA, Canada have cheap SMS fees via Nexmo.
+
+    * The UK is enabled for testing behind a UI "easter egg".
+
+* Users that sign-up for a new account
+
+### Original Version (Present Day)
+
+Here is the user flow from the moment the user finishes registration via the form.
[(Review the control after phase 1 results come in)](https://docs.google.com/document/d/13AP3rPE4RMKjECBxJzT-OuVyU3DUC1-ggpqUttAJYM4/edit#)
+
+1. User is asked to confirm email address<br />
+<img alt="Confirm your account screen" src="./image_1.png" width=300 />
+
+2. Email prepares the user for next steps<br />
+<img alt="Sync email" src="./image_2.png" width=300 />
+
+1. Depending on where the user confirms their email address, they will see one of the following screens:
+
+    1. Confirmed from same Firefox desktop<br />
<img alt="Confirm in Firefox Desktop" src="./image_3.png" width=300 />
+
+    2. Confirmed from a second disconnect Firefox instance<br />
<img alt="Confirm in another Firefox" src="./image_4.png" width=300 />
+
+    3. Confirmed from Firefox for iOS<br />
<img alt="Confirm in Firefox for iOS" src="./image_5.png" width=300 />
+
+### Variation A
+
+Here is the user flow from the moment the user finishes registration via the form.
+
+1. User is asked to confirm email address <br />
+<img alt="Confirm your account screen" src="./image_1.png" width=300 />
+
+2. Email prepares the user for next steps <br />
+<img alt="Sync email" src="./image_2.png" width=300 />
+
+3. Users that confirm on the same desktop device will see the following screen. **This screen is the main difference with the control.**<br />
+<img alt="Send Firefox to your smartphone screen" src="./image_9.png" width=300 />
+
+    * SMS Message: "As requested, here is your link to install Firefox on your mobile device: moz.la/vwxyz “
+
+### Implementation Plan
+
+#### Sending SMSs
+
+The auth server will be responsible for sending SMS. The content server will send a POST request to the auth server with the user’s phone number.
+
+#### auth server (pbooth)
+
+New endpoints are needed on the auth-server to check whether SMS sending is enabled and to send SMS messages.
+
+##### POST /sms
+
+Send SMS message identified by `messageId` to `phoneNumber`.
+
+###### Request POST data
+
+```
+sessionToken,
```

Comment:
You probably want to remove this line, the session token is in the header not the body. (the original error was mine when making notes in the Google Doc, sorry)